packages/web/src/components/nav/desktop/LeftNav.tsx

```diff
@@ -115,26 +114,47 @@ const LeftNav = (props: NavColumnProps) => {
   }, [])
 
   const profileCompletionMeter = (
-    <div className={styles.profileCompletionContainer}>
+    <Flex justifyContent='center'>
       <ConnectedProfileCompletionPane />
-    </div>
+    </Flex>
   )
 
   const navLoaded =
     accountStatus === Status.SUCCESS || accountStatus === Status.ERROR
 
   return (
-    <nav id='leftNav' className={styles.leftNav}>
+    <Flex
+      backgroundColor='surface1'
+      borderRight='default'
+      as='nav'
+      id='leftNav'
+      direction='column'
+      h='100%'
+      w='100%'
+      css={{
+        userSelect: 'none',
+        overflow: 'visible'
+      }}
+    >
       {isElectron ? <RouteNav /> : null}
       <NavHeader />
 
-      <div
+      <Flex
+        direction='column'
+        w='100%'
+        flex={1}
         ref={navBodyContainerMeasureRef}
-        className={cn(styles.leftNavContent, {
-          [styles.show]: navLoaded,
-          [styles.dragScrollingUp]: dragScrollingDirection === 'up',
-          [styles.dragScrollingDown]: dragScrollingDirection === 'down'
-        })}
+        css={{
+          boxShadow:
+            dragScrollingDirection === 'up'
+              ? 'inset 0px 8px 5px -5px var(--tile-shadow-3)'
```

Comment:
So it looks like this is harmony neutral color + 0.25 opacity... Ideally this is represented in harmony some way? what do you think?

Reply:
Yeah I think several of the colors/motion variables here are legacy and design wanted to not worry too much about it until the left nav is fully redesigned?